etc/picongpu/hemera-hzdr/gv100.tpl

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
```

Comment:
Please rename this file to `fwkt_v100.tpl`. Our files have always the name of the queue.